<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>host_window</class>
 <widget class="QMainWindow" name="host_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	color: #ffe400;
	font-family: &quot;Roboto&quot;;
}

#lbl_topBar {
	background-color: #000aff;
}

#sw_nav {
	background: #000000;
}

QDialog {
	background-color: #000000;
	border: 1px solid #ffe400;
}

QLabel {
	background-color: #00000000;
}

QPushButton {
	background: #000000;
	border-color: #ffe400;
    border-width: 2px;
    border-style: solid;
    border-radius: 8px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_topBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_indicatorGps">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>8</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>NAV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_time">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>8</y>
      <width>160</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_indicatorSrv">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>8</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>SRV</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_screen_title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>331</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="sw_nav">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
